akka-stream-tests/src/test/resources/reference.conf

```diff
@@ -8,4 +8,5 @@ akka {
   akka.actor.warn-about-java-serializer-usage = false
 
   stream.materializer.debug.fuzzing-mode = on
+  stream.secret-test-fuzzing-warning-disable = 42
```

Comment:
I see. I thought that way it would be more obvious that fuzzing is only intended for testing and we could get rid of the warning/setting.

(Correction: I guess stream-testkit ISO stream-tests would be the right place.)